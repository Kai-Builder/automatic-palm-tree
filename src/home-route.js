import {Home} from './home';
import React from 'react';
import {Route} from 'react-router-dom';

// import {SecureRoute} from '@okta/okta-react';

export const HomeRoute = (props) => (
  <Route
    exact={true}
    path='/'
    render={(routeProps) => (
      <Home
        {...routeProps}
        {...props}
      />
    )}
  />
);
