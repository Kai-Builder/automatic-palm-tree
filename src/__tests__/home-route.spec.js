import {HomeRoute} from '../home-route';
import React from 'react';
import {shallow} from 'enzyme';

describe('HomeRoute', () => {
  const requiredProps = () => ({
  });
  const render = (props = requiredProps()) => shallow(<HomeRoute {...props} />);

  test('component renders', () => {
    const wrapper = render();

    expect(wrapper).toMatchSnapshot();
  });
});
