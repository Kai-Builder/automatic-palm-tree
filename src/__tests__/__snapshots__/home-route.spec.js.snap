// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeRoute component renders 1`] = `
<Route
  exact={true}
  path="/"
  render={[Function]}
/>
`;
